drucker/drucker_worker.py

```diff
@@ -72,6 +74,12 @@ def get_type_input(self) -> Enum:
     def get_type_output(self) -> Enum:
         return self.__type_output
 
+    @abstractmethod
+    def parse_eval_data(self, file_path: str) -> Generator[EvaluateData, None, None]:
+        """parse file uploaded from dashboard
+        """
+        raise NotImplemented()
```

Comment:
`parse_eval_data` will be used in [EvaluationResult](https://github.com/rekcurd/drucker-grpc-proto/blob/master/drucker.proto#L58) as well.

Reply:
Could you explain?
My understanding is that the data format is different between `EvaluationData` and `EvaluationResult`.

Reply:
Oh, it is very confusing.
`EvaluateResult` I mentioned is a gPRC protocol, but a model, and `EvaluationResultResponse` contains input and label data, so `EvaluateData` will be used to make `EvaluationResultResponse`

Reply:
Sorry I cannot figure it out yet.

There are three types of data; `EvaluateData` as an input, `EvaluationResult` and `EvaluationDetails` as an output. These three data will be stored. Is my understanding OK?

So, how do we use `parse_eval_data` method? I think these three data have a different format.

Reply:
@keigohtr 
I removed parse_eval_data to make `Drucker` class more understandable in https://github.com/rekcurd/drucker/pull/20/commits/e8548692be2ffcb06184bdceb58c5a193f8614e7